packages/api-client/src/main/team/legalhold/LegalHoldAPI.ts

```diff
@@ -0,0 +1,108 @@
+/*
+ * Wire
+ * Copyright (C) 2019 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {AxiosRequestConfig} from 'axios';
+
+import {HttpClient} from '../../http';
+import {TeamAPI} from '../team/TeamAPI';
+import {LegalHoldData} from './LegalHoldData';
+import {NewLegalHoldData} from './NewLegalHoldData';
+
+class LegalHoldAPI {
+  constructor(private readonly client: HttpClient) {}
+
+  static URL = {
+    APPROVE_LEGAL_HOLD: 'approve',
+    LEGAL_HOLD: 'legalhold',
+    SETTINGS_LEGAL_HOLD: 'settings',
+  };
+
+  public async getMemberLegalHold(teamId: string, userId: string): Promise<void> {
+    const config: AxiosRequestConfig = {
+      method: 'get',
+      url: `${TeamAPI.URL.TEAMS}/${teamId}/${LegalHoldAPI.URL.LEGAL_HOLD}/${userId}`,
+    };
+
+    await this.client.sendJSON<void>(config);
+  }
+
+  public async deleteMemberLegalHold(teamId: string, userId: string): Promise<void> {
+    const config: AxiosRequestConfig = {
+      method: 'delete',
+      url: `${TeamAPI.URL.TEAMS}/${teamId}/${LegalHoldAPI.URL.LEGAL_HOLD}/${userId}`,
+    };
+
+    await this.client.sendJSON<void>(config);
+  }
+
+  public async postMemberLegalHold(teamId: string, userId: string): Promise<void> {
```

Comment:
There is `postMemberLegalHold` and `getLegalHoldSettings`. 

We should have a uniform naming where it is either:
- postMemberLegalHold
- getSettingsLegalHold

Or
- postLegalHoldMember
- getLegalHoldSettings